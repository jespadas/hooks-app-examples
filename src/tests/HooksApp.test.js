import React from 'react';
import { HooksApp } from '../HooksApp';
import { shallow } from 'enzyme';

describe('Test on HookApp.js', () => {

    let wrapper = shallow(<HooksApp />);

    test('Must show <HookApp /> properly', () => {

        expect(wrapper).toMatchSnapshot();

    });

})