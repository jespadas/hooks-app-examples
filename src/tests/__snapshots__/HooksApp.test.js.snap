// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <HookApp /> Must show <HookApp /> properly 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
